test/SpecRunner.html

```diff
@@ -24,6 +25,7 @@
 
   <!-- include source files here... -->
   <script src="../../fast-json-patch/src/json-patch-duplex.js"></script>
+  <script src="../../../node_modules/jsonpatcherproxy/dist/jsonpatcherproxy.min.js"></script>
   <script src="../src/palindrom.js"></script>
   <script src="../src/palindrom-dom.js"></script>
 
```

Comment:
I think that since we now give the bundled version, we must remove dependencies from `bower.json`, because they are most likely to be unused.

@tomalec do you agree?